src/afch.js

```diff
@@ -1,19 +1,9 @@
 // <nowiki>
 ( function () {
-	// Check that we're in the right namespace and on the right page
+	// Check that we're in the right namespace
+	// In the gadget, this is checked by ResourceLoader and not needed.
```

Comment:
technically, it's the Gadgets extension that does conditional load checks like namespaces. RL by itself doesn't check anything